streams/src/test/java/org/apache/kafka/streams/integration/NamedTopologyIntegrationTest.java

```diff
@@ -64,6 +64,7 @@
 import java.util.LinkedList;
 import java.util.Map;
 import java.util.Queue;
+import org.junit.Ignore;
```

Comment:
That is unused now. You probably get a spotbug error.

Reply:
Done